src/Pages/miyoungshin/Main/Main.scss

```
@@ -0,0 +1,304 @@
+@import '../styles/mixins';
+@import '../Styles/variables';
+@import '../../../Style/Common.scss';
+@import '../../../Style/Reset.css';
+
+.mainSection {
+  position: absolute;
+  top: 54px;
+  left: 0;
+  width: 100%;
+  background-color: $lightgray;
+  font-size: 13px;
+  article {
+    .stories {
+      @include flexSet(space-between);
+      width: 100%;
+      height: 110px;
+      overflow: hidden;
+      .story {
+        @include flexSet(center, center, column);
+        width: 100%;
+      }
+      .storyBg {
+        @include flexSet;
+        width: 66px;
+        height: 66px;
+        margin-bottom: 5px;
+        border-radius: 50%;
+        background: linear-gradient(90deg, #833ab4, #fd1d1d, #fcb045);
+        .storyProfile {
+          width: 60px;
+          height: 60px;
+          border: 3px solid white;
+          border-radius: 50%;
+        }
+      }
+      .storyId {
+        font-size: 12px;
+      }
+    }
+
+    .feeds {
+      width: 100vw;
+      margin-top: 15px;
+      border-top: $gray-border;
+      .feedHead {
+        @include flexSet(space-between);
+        height: 60px;
+        width: 100%;
+        padding: 20px;
+        border-bottom: $gray-border;
+        background-color: #ffffff;
+        .account {
+          @include flexSet(space-between);
+          div {
+            padding-left: 10px;
+            font-weight: bolder;
+          }
+          img {
+            width: 32px;
+            height: 32px;
+            border-radius: 50%;
+          }
+        }
+        .accountMore {
+          font: {
+            size: 20px;
+            weight: 600;
+          }
+          text-align: right;
+          cursor: pointer;
+        }
+      }
+      .feedBody {
+        img {
+          max-width: 100%;
+        }
+        .feedReaction {
+          padding: 15px;
+          margin-bottom: 10px;
+          .floatLeft {
+            @include main-feed-react-icon;
+            float: left;
+          }
+          .floatRight {
+            @include main-feed-react-icon;
+            float: right;
+          }
+        }
+      }
+
+      .feedInfo {
+        padding: 0 20px;
+        .feedLikes {
+          clear: both;
+          padding: 10px 0;
+          font-weight: bolder;
+        }
+        .feedContents {
+          padding: 5px 0;
+          .myAccountId {
+            font-weight: bolder;
+          }
+        }
+        .feedCommentsList {
+          padding: 5px 0;
+          ul {
+            padding-left: 0;
+            list-style: none;
+            li {
+              @include flexSet(space-between);
+              .commentId {
```

```suggestion
        .feedCommentsList {
          padding: 5px 0;
          
          ul {
            padding-left: 0;
            list-style: none;
            
            li {
              @include flexSet(space-between);
              .commentId {
```
새로운 셀렉터 앞에서는 한줄 띄워주시는게 가독성 측면에서 좋을거 같아요!

---

이후에는 한 줄 띄어 작성하겠습니다^^